main.py

```diff
@@ -482,12 +524,26 @@ def check_frontend(self):
             
             cookies = dict(cookies=valid_token)
             response = requests.get(url_dashboard, cookies=cookies)
-            assert (response.status_code == 200)
-            print("Dashboard page with cookies - successfully connected")
```

Comment:
I would say with and without `token` instead of `cookies` - it's a bit confusing